### PR TITLE
[WIP] feat: add admin command

### PR DESCRIPTION
**Pull request by Sqrl34**
command to change vehicles anywhere and for free.

### CODE REVIEW

**Comment by alberttheprince** [comment]
@Jorn08 This is somewhat ready, tested by @Sqrl34 and working in a QB server. 

This would be helpful for admin menus to allow admins to trigger the menu. Could you review/give sqrl some feedback?

File: client/menus/main.lua [hunk 1]
```diff
@@ -128,10 +128,6 @@ end
 menu.onClose = function()
     inMenu = false
     stopDragCam()
-    lib.showTextUI('Press [E] to tune your car', {
```
Comment (Jorn08) [inline client/menus/main.lua:131]: Why is this removed?

File: server.lua [hunk 2]
```diff
@@ -89,7 +107,19 @@ end
...
+    currentAdmins[src].admin = false
     if IsVehicleOwned(vehicleProps.plate) then
         MySQL.update.await('UPDATE player_vehicles SET mods = ? WHERE plate = ?', {json.encode(vehicleProps), vehicleProps.plate})
     end
 end)
+
+--Commands
+lib.addCommand('admincustoms', {
+    help = 'Toggle customs menu for admins',
+    restricted = 'group.admin',
```
Comment (Jorn08) [inline server.lua]: Should this not be `'customs.admin'`?
```suggestion
    restricted = 'customs.admin',
```

File: client/menus/main.lua (hunk 1 above)
Comment (Sqrl34) [reply client/menus/main.lua:131]: it was removed because when an admin finishes modifying their car it shows up and can not be removed unless you go to  the mechanic shop. Simple fix was to just remove it because players can still press E to access the shop though the text isn't showing and admins don't have the text on their screen.

File: server.lua
Comment (alberttheprince) [inline server.lua:1]: Is this an acceptable fix @Jorn08?

Comment (Jorn08) [reply server.lua:1]: There is a little bit of overhead added because of the server callback when the menu closes, but if the added delay isn't noticeable it should be fine. Have not tested it myself

**Review by Jorn08 — APPROVED** [review]

**Comment by Jorn08** [comment]
Bump the version as well
